Source/WebKit/UIProcess/Automation/mac/WebAutomationSessionMac.mm

```diff
@@ -253,20 +254,7 @@ static WebMouseEventButton automationMouseButtonToPlatformMouseButton(MouseButto
 
 OptionSet<WebEventModifier> WebAutomationSession::platformWebModifiersFromRaw(unsigned modifiers)
```

Comment:
Can we get rid of this function? Why do we need a platform-independent “raw”?

Reply:
Yes, I think so.  I see it's private to `WebAutomationSession`, but I need to rework the GTK port as well:

```
Source/WebKit/Shared/gtk/WebEventFactory.cpp
Source/WebKit/UIProcess/Automation/gtk/WebAutomationSessionGtk.cpp
```

I could also just make `WebAutomationSession::platformWebModifiersFromRaw()` an inline method on macOS.

Reply:
> I could also just make `WebAutomationSession::platformWebModifiersFromRaw()` an inline method on macOS.

To do this, I need to make `Source/WebKit/Shared/Mac/WebEventFactory.h` safe to include in C++ source.  The next patch will do this, so I want to wait on this change.